src/main/java/mat/client/shared/MessageDelegate.java

```diff
@@ -967,10 +967,17 @@ public String getUnusedIncludedLibraryWarning(String name) {
     }
 
     public String getUnusedFhirElementsWarning(String name) {
-        return "You have included libraries, codes or value sets that are unused. " +
-                "In order to version " + name + ", " +
-                "these must be removed. Select Continue to have the MAT remove these unused elements or Cancel " +
-                "to stop the version process.";
+        return "There are included libraries, codes or value sets that are unused in measure " + name + ". " +
+                "Select Continue to have the MAT remove these unused elements, " +
+                "Keep to retain the unused elements " +
+                "or Cancel to stop the version process.";
```

Comment:
Do we need to include 'DRCs' in the list of elements that might be unused?
The story has AC - Change the prompt to "You have unused libraries, values sets, code systems or and/or direct reference codes. Do you want them removed?"

Reply:
That's somewhat covered by the generic "codes". Nevertheless, you bring up a good point and that we need to update the AC and/or the dialog message to match.